.openpublishing.redirection.json

```diff
@@ -1904,6 +1904,10 @@
             "source_path": "docs/csharp/programming-guide/classes-and-structs/classes.md",
             "redirect_url": "/dotnet/csharp/fundamentals/types/classes"
         },
+	{
+            "source_path": "docs/csharp/programming-guide/classes-and-structs/finalizers.md",
+            "redirect_url": "/dotnet/csharp/programming-guide/classes-and-structs/destructors"
```

Comment:
`destructors` should be the `source_path` file, and `finalizers` should be the `redirect_url` file.

Reply:
@gewarren, unfortunately it seems not.